frontend/app/components/shared/Filters/FilterAutoComplete/AutocompleteModal.tsx

```
@@ -0,0 +1,190 @@
+
+      <div
+        className={'flex flex-col gap-2 overflow-y-auto py-2'}
+        style={{ maxHeight: 200 }}
+      >
+        {sortedOptions.map((item) => (
+          <div
+            onClick={() => onSelectOption(item)}
+            className={
+              'cursor-pointer w-full py-1 hover:bg-active-blue rounded px-2'
+            }
+          >
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)